Harvest.Web/Controllers/Api/RequestController.cs

```diff
@@ -119,14 +119,44 @@ public async Task<ActionResult> Approve(int projectId, [FromBody] RequestApprova
             }
 
             await _historyService.QuoteApproved(project.Id, model.Accounts);
- 
+
             await _dbContext.SaveChangesAsync();
 
             await _emailService.QuoteApproved(project);
 
             return Ok(project);
         }
 
+        [HttpPost]
+        [Authorize(Policy = AccessCodes.PrincipalInvestigator)]
+        public async Task<ActionResult> RejectQuote(int projectId, [FromBody] QuoteRejectionModel model)
+        {
+            var project = await _dbContext.Projects.SingleAsync(p => p.Id == projectId);
+
+            var currentUser = await _userService.GetCurrentUser();
+
+            var ticketToCreate = new Ticket();
+            ticketToCreate.ProjectId = projectId;
+            ticketToCreate.Name = "Quote rejected";
+            ticketToCreate.Requirements = model.Reason;
+            ticketToCreate.DueDate = null;
+            ticketToCreate.CreatedById = currentUser.Id;
+            ticketToCreate.CreatedOn = DateTime.UtcNow;
+
+            await _dbContext.Tickets.AddAsync(ticketToCreate);
+            await _historyService.TicketCreated(project.Id, ticketToCreate);
+
+            project.Status = Project.Statuses.QuoteRejected;
+
+            await _historyService.QuoteRejected(project.Id, model.Reason);
+
+            await _dbContext.SaveChangesAsync();
+
+            await _emailService.NewTicketCreated(project, ticketToCreate);
```

Comment:
@srkirkland There should be a quote rejected endpoint.

Reply:
@jSylvestre should i do that instead of doing a ticket even though a ticket's created?  It would probably make more sense than doing both i suppose.

Reply:
@jSylvestre Let's do that in a future issue -- if we move over to that method we should definitely (a) show the actual reason it was rejected and (b) probably link to the project page or ticket for it.

Reply:
Up to you. Possibly I can change the quote rejected email to have a link to the ticket?
Or just don't use the quote rejected, but it does stand out as a significant email.